### PR TITLE
Made Orbits PWA! 🚀

### PR DESCRIPTION
**Pull request by RajvirSingh1313**
1. Made manifest.json
2. Made next.config.js
3. Installed new package called ``next-pwa``

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        "purpose": "any maskable"
+      },
+      {
+        "src": "favicon.png",
+        "sizes": "512x512",
+        "type": "image/png",
+        "purpose": "any maskable"
+      }
+    ],
+    "display": "standalone",
```
Comment (AadityaMunjal) [inline public/manifest.json:27]: Weren't we doing fullscreen? 
```suggestion
    "display": "fullscreen",
```

Comment (RajvirSingh1313) [reply public/manifest.json:27]: I don't let me just research a bit about display types

Comment (RajvirSingh1313) [reply public/manifest.json:27]: https://superpwa.com/doc/web-app-manifest-display-modes/

I found this article and I think standalone is the right one full screen one will be be trouble some

Comment (AadityaMunjal) [reply public/manifest.json:27]: Okay thanks

**Review by AadityaMunjal — APPROVED** [review]

**Comment by AadityaMunjal** [comment]
Try running: `npm install --package-lock-only` @RajvirSingh1313

**Comment by RajvirSingh1313** [comment]
> npm install --package-lock-only

like would i need to commit it again, and then push the changes to my repo and then will they automatically add to the pr?

**Comment by AadityaMunjal** [comment]
> like would i need to commit it again, and then push the changes to my repo and then will they automatically add to the pr?

Yes, your repo's main branch

**Comment by RajvirSingh1313** [comment]
> > like would i need to commit it again, and then push the changes to my repo and then will they automatically add to the pr?
> 
> Yes, your repo's main branch

Okay i am gona do it right now

**Comment by RajvirSingh1313** [comment]
I am gona create new pull request

**Pull request closed without merging**